test/e2e/cross-account/smoke.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
Place this file in the `test/e2e/s3` directory and call it `cross-account.sh` :) That way, when we run the S3 e2e tests, we'll run the CARM tests.

Reply:
++